hexrdgui/main_window.py

```diff
@@ -2,9 +2,9 @@
 from pathlib import Path
 import shutil
 import tempfile
+from functools import partial
```

Comment:
Can this be alphabetized too? So `functools` should be the first import before `os`.